templates/candidate/components/_complementary-questions.html.twig

```diff
@@ -1,6 +1,6 @@
 <div class="row questions my-5" id="candidateComplementaryQuestions">
 	<div class="card card-questions p-4 col-md-6 offset-md-3">
-		{% if not is_granted("ROLE_ADMIN") %}
+		{% if not is_granted("ROLE_ADMIN") and not is_granted("ROLE_COMPANY") %}
```

Comment:
ça marche pas si tu met plutot 
		{% if not is_granted("ROLE_ADMIN") and is_granted("ROLE_CANDIDATE") %}
ça doit revenir au meme mais c'est plus logique non ?

Reply:
idem en dessous